lib/msal-core/src/PopTokenCommon.ts

```diff
@@ -0,0 +1,61 @@
+export enum HashAlgorithm {
+    sha256 = 'SHA-256'
+};
+
+export enum KeyGenAlgorithm {
+    rsassa_pkcs1_v15 = "RSASSA-PKCS1-v1_5"
```

Comment:
your enum casing here is different than your other, any reason?

Reply:
Laziness, mostly. Will clean these enums up on my next pass.